src/universal/modules/teamDashboard/components/ProviderRow/ProviderRowName.tsx

```diff
@@ -1,64 +1,70 @@
-import React from 'react'
```

Comment:
delete?

Reply:
Will do, I think in #2740 we remove `userCount` and `integrationCount` as I’m wanting to simplify the design and not have those icons anymore.